client-js/app/elements/labrad-plot.ts

```diff
@@ -811,3 +789,66 @@ function prettyNumber(value: number, min: number, max: number, res: number): str
   }
 }
 
+
+/**
+ * Get a color for the given value on a scale with the given min and max
+ * limits (zMin <= z <= zMax).
+ */
+function getColor(z: number, zMin: number, zMax: number) {
+  var index = 128;
+  if (zMin !== zMax) {
+    index = Math.floor(255 * (z - zMin) / (zMax - zMin));
```

Comment:
I don't see why this behavior is desired. The highest color should only occur when the color is equal to `zMax`. We don't complain that `COLOR_MAP[0]` is only used when `z == zMin`.